getting_started/setup_vm/E-ccf-deps.yml

```diff
@@ -15,6 +15,7 @@
       - llvm-7
       - expect
       - xsltproc
+      - git
     everest_ver: "mbedtls-snapshot-7"
```

Comment:
Please excuse my confusion but do we still need this?

Reply:
Yes we do, because scripts like notice-check or cimetrics call it during the CI.